_additional/fakeClient/lib/ClientHandler.js

```diff
@@ -0,0 +1,138 @@
+const DataGenerator = require('./DataGenerator');
+const request = require('request-promise');
+
+const MESSAGE_DELAY_MIN = 1000;
+const MESSAGE_DELAY_MAX = 5000;
+
+class ClientHandler {
+	constructor(address) {
+		this._address = `http://${address}`;
```

Comment:
Nie moglibysmy podawac url od razu z protokołem z pliku?

Reply:
Moglibyśmy, ale plik zaczynający każdą linię od `http://` będzie brzydki i w sumie nadmiarowy - boli mnie to. :D

Reply:
Ja jednak bym to wyrzucił do pliku. Niech ktoś jeszcze zabierze głos

Reply:
Moim zdaniem tak jest ok, jak dasz do konfiguracji to jeszcze ktoś wpadnie na pomysł wpisania https:P

Reply:
Ja bym nie trzymal w konfiguracji tylko podawal linki z pliku wraz z protkołem z ktorego bierzemy hosta